frontend/src/pages/Match/Timer.js

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from 'react';
+
+function Timer() {
+  const DURATION = 30;
```

Comment:
Possible optimization for rendering could be putting the const global variable outside of functional component, so they don't get re-rendered again when state change.

I would suggest doing this especially this component re-rendered every second, so it's good to do this although small.